Geometry/HGCalMapping/python/hgcalmapping_cff.py

```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
+
+def customise_hgcalmapper(process,
+                          modules = 'Geometry/HGCalMapping/data/ModuleMaps/modulelocator_test.txt',
+                          sicells = 'Geometry/HGCalMapping/data/CellMaps/WaferCellMapTraces.txt',
+                          sipmcells = 'Geometry/HGCalMapping/data/CellMaps/channels_sipmontile.hgcal.txt'):
+
+    process.load('Geometry.HGCalMapping.hgCalMappingESProducer_cfi')
+    process.hgCalMappingESProducer.modules = cms.FileInPath(modules)
+    process.hgCalMappingESProducer.si = cms.FileInPath(sicells)
+    process.hgCalMappingESProducer.sipm = cms.FileInPath(sipmcells)
+
+    if not hasattr(process, 'ProcessAcceleratorCUDA'):
+        print('Looks like Configuration.StandardSequences.Accelerators_cff was not yet loaded...loading')
```

Comment:
Please do not `print` in a ptyhon configuration, because the message will end up in the configurations created by _e.g._ `edmConfigDump`.

Reply:
oh !